docs/install/iam_policy.json

```diff
@@ -236,6 +236,13 @@
                 "elasticloadbalancing:ModifyRule"
             ],
             "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
```

Comment:
let's not add to default policy since customers don't need this by default.
we can specify in documentation that `tag:GetResources` is needed when enable the feature flag